src/FESpaces.jl

```diff
@@ -33,8 +37,16 @@ function FESpaces.zero_free_values(f::DistributedFESpace)
   fill!(vec,zero(eltype(vec)))
 end
 
+function FESpaces.zero_dirichlet_values(f::DistributedFESpace)
```

Comment:
I am not sure if it is correct. What is the type of `vec` ? 

I think that the "tricky" part regarding Dirichlet dofs is that we never generate a global numeration for them. Thus, the  queries related with dirichlet dofs, should to return an `AbstractPData` object that wraps the corresponding serial Gridap call. They cannot return global objects that would require to generate a global numeration for them.

Another option would be to create a global numeration for Dirichlet dofs. It would be more elegant because the rationale for free and Dirichlet would be equivalent, but this is an extra cost not needed in general.

Reply:
Removed in fa9873ea6a8b6bebe8a481016138297fc0687dfc (to be re-implemented in another PR if needed)